index.html

```diff
@@ -235,19 +228,13 @@ <h2>With JOSE</h2>
         <h2>Securing JSON-LD Verifiable Credentials with JOSE</h2>
         <p>
           This section details how to use JOSE to secure verifiable credentials conforming
-          to the [[VC-DATA-MODEL]].
-        </p>
-        <p>[[rfc7515]] MAY be used to secure this media type.</p>
-        <p>
-          The <code>typ</code> parameter SHOULD be <code>vc+ld+json+sd-jwt</code>
+          to [[VC-DATA-MODEL-2.0]].
         </p>
         <p>
-          When present, the <code>cty</code> SHOULD be
-          <code>vc+ld+json</code>
-        </p>
-        <p>
-          See <a data-cite="rfc7515#section-4.1.10">Common JOSE Header
-            Parameters</a>
+          [[RFC7515]] MAY be used to secure this media type.
+          The <code>typ</code> header parameter SHOULD be <code>vc+ld+json+sd-jwt</code>.
```

Comment:
```suggestion
          The <code>typ</code> header parameter SHOULD be <code>sd-jwt</code>.
```

Reply:
isn't the original needed to show that it's a vc+ld payload represented in an sd-jwt?